services/api/src/typeDefs.js

```diff
@@ -340,6 +341,10 @@ const typeDefs = gql`
     Which groups are directly linked to project
     """
     groups: [GroupInterface]
+    """
+    Metadata key/values stored against a project
+    """
+    metadata: String
```

Comment:
there is a JSON field in GraphQL, see example:
https://github.com/amazeeio/lagoon/blob/8d95f023c12139b3672ce38582f5b0ebe9481cf2/services/api/src/typeDefs.js#L614
I think we can use that one?